comm/src/main/scala/coop/rchain/comm/comm.scala

```diff
@@ -1,10 +1,11 @@
 package coop.rchain.comm
 
-import scala.util.Try
+trait CommError
```

Comment:
Please use `sealed trait`, this prevents someone to extend your ADT and helps compiler to check that your pattern matching expressions is exhaustive.